packages/wizard/src/wizard.tsx

```diff
@@ -0,0 +1,83 @@
+import {
+  chakra,
+  forwardRef,
+  HTMLChakraProps,
+  omitThemingProps,
+  StylesProvider,
+  ThemingProps,
+  useMultiStyleConfig,
+} from "@chakra-ui/system"
+import { cx } from "@chakra-ui/utils"
+import * as React from "react"
+
+export interface WizardProps extends HTMLChakraProps<"div">, ThemingProps {
+  activeStep: number
+  orientation?: "vertical" | "horizontal"
+  labelOrientation?: "vertical" | "horizontal"
+  isLoading?: boolean
```

Comment:
`isLoading` and `isError` is something to avoid in component API design in general for [more insights](https://kentcdodds.com/blog/stop-using-isloading-booleans).

TL;DR `isLoading` and `isError` are mutual exclusive (can not occur at the same time) so the state of a step should be reflected as an enum instead.


But long story short I would not support this anyway (at start).

I would suggest to start things off with `activeStep: number` and `orientaiton: "vertical" | "horizontal"`

Reply:
Cool, sounds good I will remove these states for now 👍🏼